src/test/java/org/broadinstitute/dsde/consent/ontology/datause/DataUseMatcherTest.java

```diff
@@ -271,11 +270,7 @@ private void assertNegative(DataUse purpose, DataUse dataset) {
     private ImmutablePair<Boolean, List<String>> matchPurposeAndDataset(DataUse purpose, DataUse dataset) {
         DataUseMatcher matcher = new DataUseMatcher();
         matcher.setAutocompleteService(autocompleteService);
-        try {
```

Comment:
Why did you remove the try-catch error handling here? Does it get handled in another place?

Reply:
Yes, exactly, line 43 of `DataUseMatcher` is the method signature which no longer throws an exception. Inside the method, we handle the error case.

Reply:
sounds good!